.github/workflows/lint-daggerized.yml

```diff
@@ -0,0 +1,36 @@
+name: lint-daggerized
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+  # Enable manual trigger for easy debugging
+  # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+
+jobs:
+  engine:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: sipsma/dagger-experiments@v5
+      - run: dagger do --debug --project demos/go-entrypoints/dagger.json ci '${{ toJson(github.event) }}'
```

Comment:
I was just experimenting with event ingestion here, this target doesn't actually exist anymore. When I finish migrating the provisioning tests to an entrypoint there will be a real example of this since the payload will be needed